test/benchmarks/netperf/netperf-rc.yaml.in

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    app: {{ podname }}
+    
+  name: {{ podname }}
+  namespace: netperf
+spec:
+  replicas: 1
+  selector:
+    app: {{ podname }}
+  template:
+    metadata:
+      labels:
+        app: {{ podname }}
+    spec:
+      # Force this pod onto the specified node
+      nodeSelector:
```

Comment:
I was following this documentation to assign a pod created by an rc to a node. I can implement an easier way - could you send me a pointer to the mechanism ?

http://releases.k8s.io/release-1.2/docs/user-guide/node-selection/README.md